src/providers/ogr/qgsogrprovider.h

```diff
@@ -212,6 +298,49 @@ class QgsOgrProvider : public QgsVectorDataProvider
     //! String used to define a subset of the layer
     QString mSubsetString;
 
+    /** MSubLayerList entry that was actually used to open the layer
```

Comment:
Thanks to the spell checker, most of the Gdal 1.* comments in the documentation should now be gone.